From f487de038cb9cd149cbade42b9a69ec5d10bbca5 Mon Sep 17 00:00:00 2001
From: Emmanuel Jillela <emmanuel.jillela@intel.com>
Date: Thu, 19 Nov 2015 13:37:12 -0700
Subject: [PATCH 237/639]   Title: Remove the unnecessary printks in HW Mutex
 ISR   Issue: <HSD:244656>   IssueType: <BUG>   Description: <Removing the
 printk in HW mutex ISR used while debugging.>   Reviewers: <ejillela>

---
 kernel/hwmutex.c | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/kernel/hwmutex.c b/kernel/hwmutex.c
index f7d9e2b..26f92d0 100644
--- a/kernel/hwmutex.c
+++ b/kernel/hwmutex.c
@@ -108,20 +108,14 @@ static irqreturn_t hw_mutex_isr(int irq, void *dev_id)
                     if (hmutex_ops->is_locked(hmutex)) {
                         atomic_set(&hmutex->status, HW_MUTEX_LOCKED);
                         if (likely(hw_mutex_get_owner(hmutex))) {
-                            printk("HW MUTEX ISR: acquiring lock\n");
                             wake_up_process(hmutex->owner->task);
                         }
                         else {
                             /* Nobody need the MUTEX, just unlock it to avoid deadlock */
-                            printk("HW MUTEX ISR: Nobody waiting\n");
                             hmutex_ops->unlock(hmutex);
                         }
-                    } else {
-                        printk("HW MUTEX ISR: IS LOCKED current master is not holdig the lock\n");
-                    }
-                } else {
-                    printk("HW MUTEX ISR: status is not requesting\n");
-                }
+                    } 
+                } 
                 spin_unlock(&hmutex->irq_lock);
             }
             break;
-- 
2.10.1

